src/parallel/parallel.hpp

```diff
@@ -194,10 +199,10 @@ class ParallelRobinHoodHashTable {
     find(KeyType key);
 
 private:
-    std::vector<ParallelBucket> buckets_;
+    std::vector<ParallelBucket> buckets_ = std::vector<ParallelBucket>(1);
```

Comment:
This is not how you initialize a vector of length 1, AFAIK.

@arctic-marmoset ?

Reply:
@thedavidchu This does actually work. `std::vector(N)` default-initialises N elements.